wheel1_controller:
  type: effort_controllers/JointVelocityController
  joint: joint1_1
  pid:
    p: 1000
    d: 10
    i: 0.5
    i_clamp: 0.5

wheel2_controller:
  type: effort_controllers/JointVelocityController
  joint: joint2_1
  pid:
    p: 1000
    d: 10
    i: 0.5
    i_clamp: 0.5

wheel3_controller:
  type: effort_controllers/JointVelocityController
  joint: joint3_1
  pid:
    p: 1000
    d: 10
    i: 0.5
    i_clamp: 0.5

wheel4_controller:
  type: effort_controllers/JointVelocityController
  joint: joint4_1
  pid:
    p: 1000
    d: 10
    i: 0.5
    i_clamp: 0.5